### PR TITLE
Allowing fixing/constraining spotcon to an input model file

### PR DESCRIPTION
**Pull request by taylorbell57**
This PR allows users to fix or constrain the starspot contrast parameter, `spotcon`, to values from an input model file in the same way that we allow the input of limb-darkening coefficients. I've found that the spot parameters can be fairly well constrained by broadband fits, but for spectroscopic fits I fix the spot size and location parameters to the results from the broadband fit and I use a ratio of PHOENIX models that matches the fitted `spotcon` value from my broadband fit. Since `fleck` only allows a single contrast for all spots, the `spotcon_file` for `fleck` will only use the first column, but if the user uses `starry` then there can be a separate column for each spot (if the user wants the same contrast for all spots, they can provide just a single column file and the code will just duplicate that for the other spots).

### CODE REVIEW

File: docs/source/ecf.rst [hunk 1]
```diff
@@ -945,15 +945,31 @@ The following three parameters control the use of pre-generated limb darkening c
...
 
 ld_file_white
 ^^^^^^^^^^^^^
-The same type of parameter as ld_file, but for the limb-darkening coefficients to be used for the white-light fit. This parameter is required if ld_file is not None and any of your EPF parameters are set to white_free or white_fixed. If no parameter is set to white_free or white_fixed, then this parameter is ignored.
+The same type of parameter as ``ld_file``, but for the limb-darkening coefficients to be used for the white-light fit. This parameter is required if ``ld_file`` is not None and any of your EPF parameters are set to ``white_free`` or ``white_fixed``. If no parameter is set to ``white_free`` or ``white_fixed``, then this parameter is ignored.
+
+recenter_ld_prior
```
Comment (kevin218) [inline docs/source/ecf.rst:958]: Is there any reason we can't re-center priors when using the Exotic-ld coefficients?

**Review by kevin218 — CHANGES_REQUESTED** [review]

File: docs/source/ecf.rst (hunk 1 above)
Comment (taylorbell57) [reply docs/source/ecf.rst:958]: That's exactly what this parameter was made to do. If you're fixing your LD parameters to any LD model, then you don't need this parameter. But if you want to use a Gaussian prior around any LD model, then you need to set this parameter to True

Comment (taylorbell57) [reply docs/source/ecf.rst:958]: Oh I see what you're saying now - let me fix that

Comment (taylorbell57) [reply docs/source/ecf.rst:958]: Okay, should be good now. Sorry about the confusion - reading/writing this from my phone on the airplane

**Review by kevin218 — APPROVED** [review]
Enjoy your trip. Happy holidays!

**Pull request merged**